src/components/common/Job-Full-Details/jobFullDetails.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import Dialog from '@material-ui/core/Dialog';
+import MuiDialogTitle from '@material-ui/core/DialogTitle';
+import MuiDialogContent from '@material-ui/core/DialogContent';
+import MuiDialogActions from '@material-ui/core/DialogActions';
+import Typography from '@material-ui/core/Typography';
+import { WithStyles, withStyles } from '@material-ui/core/styles';
+import { Advertisement } from '../../../types/Advertisements';
+import SaveJobButton from '../Job/saveJobButton';
+import JobTitle from '../Job/jobTitle';
+import styles from '../Job/jobStyles';
+
+
+interface JobFullDetailsData {
+    isOpen: boolean;
+    setIsOpen: (shouldOpen: boolean) => void;
+    ad: Advertisement;
+    isFavorite: boolean;
+    isRamadView: boolean;
+    onSave: () => void;
+}
+
+type JobFullDetailsProps = JobFullDetailsData & WithStyles<typeof styles>;
+
+const JobFullDetails: React.FC<JobFullDetailsProps> = (props): JSX.Element => {
+    const {classes, isOpen, setIsOpen, ad, isRamadView, isFavorite, onSave} = props;
+
+    const handleClose = (): void => {
+        setIsOpen(false);
+    };
+
+    const renderDialogTitle = (): JSX.Element => {
```

Comment:
Nice spliting! 💯 

Regarding all render methods

Reply:
Daniel, I think your'e overusing :100:, it's more comfortable to go through a clean MR without unnecessary comments